test/integration/features/basic.feature

```diff
@@ -84,6 +84,58 @@ Feature: Basic
       172.30.1.1:5000
       """
 
+  Scenario: Deploy ruby-ex s2i image
+    Given Minishift has state "Running"
+     When executing "oc login --username=developer --password=developer" retrying 24 times with wait period of 5 seconds
+     Then stderr should be empty
+      And exitcode should equal 0
+      And stdout should contain
+      """
+      Login successful
+      """
+    Then executing "oc new-project rubys2i" retrying 3 times with wait period of 5 seconds
```

Comment:
Again, why the retry. The creation of the app should work always. If not, there is an issue. Effectively you are putting into code/test here, that it is ok for the 'oc new-project rubys2i' command to fail 2 times before it eventually will work. This is not the case